extra/prometheus/prometheus.yml

```diff
@@ -1,28 +1,15 @@
 global:
   scrape_interval:     30s # Set the scrape interval to every 30 seconds. Default is every 1 minute.
-  scrape_timeout:      30s # Set the scrape output to every 30 seconds. Default is every 1 minute.
+  scrape_timeout:      30s # Set the scrape timeout to shorter or match. Default is every 1 minute.
```

Comment:
when you say match, you should specifiy match `scrape_interval`

Reply:
Resolved in commit de6ba1b4812834ef195edb505000770802cc245b